source/file/Importer.js

```diff
@@ -110,12 +110,14 @@ class Importer {
     const minY = Math.min(...tileYs)
     const maxY = Math.max(...tileYs)
     const minX = Math.min(...tiles.map(tile => tile.position.x))
+    const yRange = maxY - minY
     return tiles.map(tile => {
       return {
         id: tile.id,
         position: {
           x: (tile.position.x - minX) + (IMPORT_TILE_MARGINS - LEFT_BIAS),
-          y: ((maxY - minY) - (tile.position.y - minY)) + IMPORT_TILE_MARGINS,
+          y: ((yRange - (tile.position.y - minY)) - (yRange % 2))
```

Comment:
Hmmm (yRange - (tile.position.y - minY)) is the same as just (maxY - tile.position.y). Also I don't fully understand the reason for taking the modulo of yRange versus maxY. But I haven't done much with the import so I'm not as familiar with it.